lib/ansible/modules/windows/win_wait_for.py

```diff
@@ -69,6 +69,7 @@
     - If C(state) is absent then it will wait until the regex does not match.
     - Defaults to a multiline regex.
     type: str
+    aliases: [ "regex", "regexp" ]
```

Comment:
Same here, `search_regex` is now an alias whereas `regex` is the proper module option.